### DISCUSSION TITLE
please help my new meetings are not showing up

### DISCUSSION THREAD

**Discussion by mmerlino90** [code4recovery/12-step-meeting-list#1273] [Q&A] [answered]
someone else in this forum also had the same issue, whenever i post a new meeting it shows up on the page for like literally 15 seconds and then you can't find it on there anymore, but it is still on the meetings page in the admin panel, it's just not showing up for users and the location is deleted :(

**Comment by joshreisner** [comment]
hi - it sounds like you're having the [same issue that palm beach county is](https://github.com/code4recovery/12-step-meeting-list/discussions/1257), that when you go to add a new meeting the location information is not saved and associated with the meeting?

here's a quick video where i demo the issue that palm beach is having specifically, please let me know if this is the same thing you're seeing? https://www.loom.com/share/8c6d720a4f39412c98ea2e73fd111252

what is your website URL, and where is it hosted? thanks

**Reply by joshreisner** [reply]
hi @mmerlino90 we're researching this issue. it appears to be limited only to sites running on wordpress.com. we believe there may be some kind of dynamic database caching layer that wordpress.com added recently, which is preventing our plugin from seeing the connection between new meetings and their locations. we'll keep investigating and will update you when we know more.

**Comment by mmerlino90** [comment]
yes it is the same issue :) our website is hosted by wordpress and link is
http://devnasouthernutah.com

**Reply by joshreisner** [reply]
hosted at wordpress.com? this could be significant because we've been unable to reproduce the issue but palm beach is also hosted there

**Reply by joshreisner** [reply]
@mmerlino90 if you could confirm if you mean that your website is hosted at wordpress.com that will help us move forward with debugging this issue, thanks

**Comment by mmerlino90** [comment]
it seems like it stopped working after the latest update about a month ago
:/

**Comment by joshreisner** [comment, marked as answer]
hi @mmerlino90 ok we have a change that we think will fix the issues on WordPress.com. if you're so inclined, you can[ read about it here](https://github.com/code4recovery/12-step-meeting-list/pull/1277), but the tl;dr is that since it's a big refactor we need to test it thoroughly before we roll it out to all users.

if you would like to help us test we would welcome that. you could download this ZIP file:  
[12-step-meeting-list.zip](https://github.com/code4recovery/12-step-meeting-list/files/13625711/12-step-meeting-list.zip) and then
on your site go to Plugins > Add New Plugin > Upload Plugin > Replace existing plugin

if you do that please let us know how it goes, but feel free not to and the update should roll out in a few days as a regular plugin update

**Comment by mmerlino90** [comment]
it seems to be working 🙂 thank you guys so much for the quick response and
quickly getting back to me to help get it fixed :-) i am the webservant of
our local NA area and had a lot of people worried about it but you guys
really came through 🙂
